ci-operator/step-registry/hypershift/mce/agent/oadp/hypershift-mce-agent-oadp-commands.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+cat <<EOF > /tmp/miniocred
+[default]
+aws_access_key_id=admin
+aws_secret_access_key=admin123
+EOF
+
+oc create secret generic cloud-credentials -n openshift-adp --from-file cloud=/tmp/miniocred
+
+cat <<EOF | oc apply -f -
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: dpa-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: minio
+          s3ForcePathStyle: 'true'
+          s3Url: 'http://virthost.ostest.test.metalkube.org:9000'
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: oadp-backup
```

Comment:
Just for you awareness, you will need to pre-create the bucket and public it in order to allow pulling for recovery stage.

Reply:
Thanks, I configured them in the MinIO step： https://github.com/openshift/release/blob/c561e312d49a66e490296c3156a366a567dd1724/ci-operator/step-registry/hypershift/mce/agent/minio/hypershift-mce-agent-minio-commands.sh#L13-L22